src/dug/core.py

```diff
@@ -42,7 +41,7 @@ def __init__(self, message, details):
 
 
 class ParserNotFoundException(Exception):
-    pass
+    ...
```

Comment:
changed this to ellipsis to be consistent with `def status(self):` at the bottom of the file